proto/xudrpc.proto

```diff
@@ -188,7 +188,7 @@ message Order {
   // The id of the peer that created this order
   int32 peer_id = 4 [json_name = "peer_id"];
```

Comment:
i guess we have no more `host_id` its just `peer_id` alone as per @sangaman's recent PR #256

Reply:
#256 wasn't merged yet. The one who gets merged after the other will have to resolve the conflict.

Reply:
You can leave the field unchanged though right? Or is there a reason this has to be changed? I do think you're right that it *should* be an int but it's gonna get dropped anyway.